### DISCUSSION TITLE
Not getting any attacks

### DISCUSSION THREAD

**Discussion by Berwynnnn** [telekom-security/tpotce#1157] [Q&A]
Hi i am completely new to honeypots and this is the first honeypot i am using to learn regarding honeypots. 
After running T-pot for an hour, there's still no attacks on my T-pot dashboard. Is there a way for me to troubleshoot why?
Thank you.

**Comment by t3chn0m4g3** [comment]
I recommend to start with #995 and then provide enough information so others are able to help.

**Comment by Berwynnnn** [comment]
Hello Sir, thank you for your reply. I apologise for being new to Honeypots but i will like to take this opportunity to learn. After looking through #995 , i am not able to find any answers that is similar to my situation. I am receiving attacks only from my local network and not any public locations. I believe the reason is due to T-pot not being exposed to the Internet? Is it possible to suggest me some commands outputs or what information i can post here for others to help me? 

I am currently running a Virtual Machine using VirtualBox and i am using Bridged Adapter for my Network Interface.

Thank you

**Reply by sengkoil** [reply]
hi,
you are using virtualbox environment in your workplace or in you home network.

if you are using from your workplace network and your company have firewall, you need to request to your network admin to do some NAT configuration and pointing to the port you want to open and to provide public IP for your honeypot server.

**Reply by alfredsecure** [reply]
Hello @sengkoil. By public IP, you meant, DHCP or Static IP?
